packages/zosfiles/src/methods/copy/Copy.ts

```diff
@@ -93,6 +103,75 @@ export class Copy {
         }
     }
 
+    /**
+     * Private function that checks if a dataset is type PDS
+    **/
+    public static async isPDS(
+        session: AbstractSession,
+        dataSetName: string
+    ): Promise<boolean> {
+        try {
+            const response = await List.dataSet(session, dataSetName, {attributes: true});
+            const dsorg = response.apiResponse.items[0].dsorg;
+            return dsorg === "POE" || dsorg === "PO";
```

Comment:
The `dsorg` for an extended PDS is `PO-E`, so we'll need to update this line for detecting PDSEs:

```suggestion
            return dsorg === "PO-E" || dsorg === "PO";
```